utils/server-side-helper/redirection/index.ts

```diff
@@ -1,130 +0,0 @@
-import { HttpNotFound } from '#clients/exceptions';
```

Comment:
I'm really glad that this file is removed. I never was really comfortable with the handling of error and redirection at this level, in the same file.